cli/remotebazel/remotebazel.go

```diff
@@ -928,6 +928,21 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 	for {
 		inRsp, executeResponse, latestErr = attemptRun(ctx, bbClient, execClient, req)
 
+		// Handle known error conditions.
+		if latestErr != nil {
+			if strings.Contains(latestErr.Error(), "failed to get snapshot") {
+				log.Warnf("The requested snapshot was not found. Failing the request...")
```

Comment:
"Failing the request" feels a bit unclear, since the client can't actively "fail" a request other than by canceling it. Did the request actually fail on the server and we're just telling the user it failed? or, what happens when this fails and can we explain what we're doing to recover?

Reply:
Changed the language a bit - let me know what you think